js/server.js

```diff
@@ -72,7 +72,7 @@ function Server (config) {
 			app.use(helmet(config.httpHeaders));
 			app.use("/js", express.static(__dirname));
 
-			let directories = ["/config", "/css", "/fonts", "/modules", "/vendor", "/translations", "/tests/configs", "/tests/mocks"];
+			let directories = ["/config", "/css", "/fonts", "/modules", "/vendor", "/translations", "/tests"];
```

Comment:
why is this still needed when moved to `/tests/mocks`?

Reply:
because in test mode you leave the path as modules, but the calendar module i use is in modules/default

so it is not found , read my post before the push

Reply:
tested your branch with `let directories = ["/config", "/css", "/fonts", "/modules", "/vendor", "/translations", "/tests/configs", "/tests/mocks"];` and it works

Reply:
oh, tests/mocks back in, yes

Reply:
have grandson til 3pm now